app/Controller/AppController.php

```diff
@@ -28,6 +28,7 @@ public function accessForbidden($withoutLayout = false, $message = '')
                 'title' => t('Access Forbidden'),
                 'no_layout' => $withoutLayout,
             )));
+            $this->response->withStatusCode(403);
```

Comment:
The status code should be set as second argument of the method html(), see https://github.com/kanboard/kanboard/blob/master/app/Core/Http/Response.php#L267